AppXManifest.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm guessing we actually need the xml version & encoding as first tag in this file (as opposed to the license header)

Reply:
Yup, that's correct. The XML fails validation in MakeAppx if the comment precedes the XML header.